docs/_packages/oauth.md

```diff
@@ -288,6 +288,23 @@ const installer = new InstallProvider({
 ```
 ---
 
+### State verification
+
+By default, this package handles generating and verifying a `state` parameter during OAuth installation. This added measure helps to mitigate the risk of [Cross-Site Request Forgery](https://datatracker.ietf.org/doc/html/rfc6749#section-10.12) and is strongly recommended. 
+
+In specific installation scenarios, such as when an Org-Wide app is installed from an admin page, state verification cannot be completed because a `state` parameter isn't provided. In this case, you can disable `state` verification via setting the `InstallProvider#stateVerification` option to `false`. In this case, the installer will no longer require that `state` be present to proceed with installation. If `state` is provided in the request url parameters, it will still be verified.
+
+```javascript
+const installer = new InstallProvider({
+  clientId: process.env.SLACK_CLIENT_ID,
+  clientSecret: process.env.SLACK_CLIENT_SECRET,
+  stateSecret: 'my-state-secret',
```

Comment:
Should developers still set stateSecret here? 

In the above comment, you mentioned that even with stateVerification: false, state parameter can be generated. But I don't think that it's easy-to-understand behavior. If the flag is false, we can skip state parameter generation.

Reply:
Great point! With the latest merged changes to the [oauth package](https://github.com/slackapi/node-slack-sdk/pull/1339), state store initialization is skipped when state verification is disabled (so implicitly state parameter generation and verification are both skipped as well). This also means that we can support removing the hard requirement for developers to set `stateSecret` in these specific cases. Hopefully that presents a more clear developer experience. 


```javascript
const installer = new InstallProvider({
  clientId: process.env.SLACK_CLIENT_ID,
  clientSecret: process.env.SLACK_CLIENT_SECRET,
  stateVerification: false,
  stateSecret: 'mysecret' // no longer required when stateVerification is false
  },
});
```